hal/src/main/native/include/hal/SPI.h

```diff
@@ -244,6 +244,19 @@ int32_t HAL_ReadSPIAutoReceivedData(HAL_SPIPort port, uint32_t* buffer,
  */
 int32_t HAL_GetSPIAutoDroppedCount(HAL_SPIPort port, int32_t* status);
 
+/**
+ * Configure the Auto SPI Stall time between reads.
+ *
+ * @param port The number of the port to use. 0-3 for Onboard CS0-CS2, 4 for
+ * MXP.
+ * @param csToSclkTicks the number of ticks to wait before asserting the cs pin
+ * @param stallTicks the number of ticks to stall for
+ * @param pow2BytesPerRead the number of bytes to read before stalling
```

Comment:
pow2BytesPerRead sounds like it's a power-of-two?  Or it has to be a power of 2?  What is the contract here?

Reply:
I'm not sure. The API was added to the labview implementation for the 470, but I couldn't figure out what changed other then just setting this API to what seem like random values to me.

Reply:
I'm okay with merging it as-is, but it would be nice to have better docs from NI for this.